src/main/resources/jenkins/plugins/extracolumns/WorkspaceColumn/column.jelly

```diff
@@ -35,7 +35,7 @@ THE SOFTWARE.
     <td>
         <j:if test="${job.hasPermission(job.WORKSPACE) and it.showWorkspace(job)}">
           <a href="${ws_url}">
-              <img src="${rootURL}/images/${iconSize}/folder.png" title="${%Workspace}" alt="${%Workspace}" border="0"/>
+              <l:icon class="icon-folder icon-md" title="${%Workspace}" alt="${%Workspace}" />
```

Comment:
Since icon size depends on the selection beneath the job list ("Icon: S M L"), ${subIconSizeClass} should be used instead of a hard-coded size ("icon-md").